ts/components/leftpane/overlay/choose-action/ContactRow.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import styled, { CSSProperties } from 'styled-components';
+import { useAvatarPath } from '../../../../hooks/useParamSelector';
+import { openConversationWithMessages } from '../../../../state/ducks/conversations';
+import { updateUserDetailsModal } from '../../../../state/ducks/modalDialog';
+import { Avatar, AvatarSize } from '../../../avatar/Avatar';
+
+type Props = { id: string; displayName?: string; style: CSSProperties };
+
+const StyledAvatarItem = styled.div`
+  padding-right: var(--margins-sm);
+`;
+
+const AvatarItem = (props: Pick<Props, 'displayName' | 'id'>) => {
+  const { id, displayName } = props;
+
+  const avatarPath = useAvatarPath(id);
+  const dispatch = useDispatch();
+  function onPrivateAvatarClick() {
+    dispatch(
+      updateUserDetailsModal({
+        conversationId: id,
+        userName: displayName || '',
+        authorAvatarPath: avatarPath,
+      })
+    );
+  }
+
+  return (
+    <StyledAvatarItem>
+      <Avatar size={AvatarSize.S} pubkey={id} onAvatarClick={onPrivateAvatarClick} />
+    </StyledAvatarItem>
+  );
+};
+
+const StyledContactRowName = styled.div`
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  /* font-weight: 600; */
```

Comment:
Do we still need this? It will resolve correctly if we want this font weight once we have merged in emoji reacts into clearnet